runtime/list_test.go

```diff
@@ -353,6 +353,39 @@ func TestListIteratorIter(t *testing.T) {
 	}
 }
 
+func TestListPop(t *testing.T) {
+	fun := newBuiltinFunction("TestListPop", func(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
```

Comment:
Whoops, yeah you're right. Similar but easier is: `append(Args{l.ToObject()}, args...)`